doc/content/tutorials/openmc_fluid.md

```diff
@@ -226,6 +226,11 @@ You can also use the XML files checked in to the `tutorials/gas_assembly` direct
 if you use these already-existing files, you will also need to download the `geometry.xml` file
 from Box; this file is large due to the saved [!ac](TRISO) geometry information.
 
+!alert note title=Density Units
```

Comment:
Please remove -- there are many of our other tutorials which use kg/m3, it does not require a special callout.